src/writer/serve.py

```
@@ -257,6 +257,20 @@ def _check_origin_header(origin_header: Optional[str]) -> bool:
 
     # Init
 
+    def _apply_feature_flags_to_templates(
+        templates: Dict[str, Any], feature_flags: List[str]
+    ) -> Dict[str, Any]:
+        """
+        Applies feature flags to the templates by removing the ones that are not enabled.
+        """
+        # Restirct API trigger by feature flag
+        if not "api_trigger" in feature_flags:
```

Ruff (0.11.9)

Test for membership should be `not in`

Convert to `not in`

(E713)